projectile.el

```diff
@@ -317,6 +317,7 @@ The topmost match has precedence."
     ".git"        ; Git VCS root dir
     ".hg"         ; Mercurial VCS root dir
     ".fslckout"   ; Fossil VCS root dir
+    "_FOSSIL_"    ; Fossil VCS root DB on Windows
```

Comment:
The only commit involving `_FOSSIL_` (according to `git log -S'_FOSSIL'`) is the one resulting from #1028, which replaced `.fossil` with `.fslckout` and `_FOSSIL_` cases in a few limited cases. The existing code that allows Fossil repos with `.fslckout` to work is from 8576b3a776f7e8554d5bf3865f03d9a1de23951e, but that change didn't take into account `_FOSSIL_`.

What it looks like to me is that Fossil support was implemented incompletely and incorrectly in 2013, and earlier this year the incorrect parts were fixed while leaving the support not quite right on Windows. If I'm missing anything, please let me know!